src/Microsoft.ApplicationInsights.AspNetCore/Logging/Implementation/ApplicationInsightsLoggerFactoryExtensions.cs

```diff
@@ -17,24 +17,23 @@ public static class ApplicationInsightsLoggerFactoryExtensions
         /// <summary>
         /// Adds an ApplicationInsights logger that is enabled for <see cref="LogLevel.Warning"/> or higher.
         /// </summary>
-        public static ILoggerFactory AddApplicationInsights(this ILoggerFactory factory,
-            TelemetryClient client)
+        public static ILoggerFactory AddApplicationInsights(this ILoggerFactory factory, IServiceProvider serviceProvider)
         {
-            return factory.AddApplicationInsights(client, LogLevel.Warning);
+            return factory.AddApplicationInsights(serviceProvider, LogLevel.Warning);
         }
 
         /// <summary>
         /// Adds an ApplicationInsights logger that is enabled for <see cref="LogLevel"/>s of minLevel or higher.
         /// </summary>
         /// <param name="factory"></param>
-        /// <param name="client">The instance of <see cref="TelemetryClient"/> to use for logging.</param>
+        /// <param name="serviceProvider">The instance of <see cref="IServiceProvider"/> to use for logging.</param>
```

Comment:
The param comment here and below should be updated to reflect the new type being passed since the service provider isn't being used for logging.